### ISSUE TITLE
Invalid anchor link to Server Events

### ISSUE THREAD

**Issue by JustinBeaudry** [hapijs/hapijs.com#580]
<img width="1399" alt="screen shot 2018-04-13 at 3 14 08 pm" src="https://user-images.githubusercontent.com/686771/38759950-5874f460-3f2d-11e8-9cbd-c00728b02b2a.png">
Links to https://hapijs.com/api#server-events when it is actualy https://hapijs.com/api#-serverevents. My proposal is just to fix the anchor to what is in the link in the image above.

**Comment by JustinBeaudry** [comment]
@hueniverse @nlf What is the process for getting the docs updated after having an update to the documentation merged (for example:  in hapi)?

**Comment by nlf** [comment]
Currently it’s a manual process, the doc gets cached for a very long time the first time it’s rendered. We don’t update docs for past versions since the docs are rendered for the tag. You would have to move the tag in git and then flush the cached version

**Comment by nlf** [comment]
Maybe having an api doc entry for what currently resides in master isn’t a bad idea though

**Comment by nlf** [comment]
That’s correct, yes. The original idea was to make it so you only see docs for published versions so as to avoid confusion about the docs being inaccurate
